### PR TITLE
[test] Migrate tool checks to hpctools.git

### PR DESCRIPTION
**Pull request by jgphpc**
Will fix https://jira.cscs.ch/browse/UES-675

### CODE REVIEW

File: cscs-checks/tools/profiling_and_debugging/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+This directory contains scripts to test HPC debugging and performance tools on
+Piz Daint using ReFrame. More checks are available here:
+http://github.com/eth-cscs/hpctools
```
Comment (vkarak) [inline cscs-checks/tools/profiling_and_debugging/README.md]: What I have not understood is which tools test go to the hpctools and which here.

File: cscs-checks/tools/profiling_and_debugging/roofline/berkeley-ert-serial.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
+import os
```
Comment (vkarak) [inline cscs-checks/tools/profiling_and_debugging/roofline/berkeley-ert-serial.py]: Please our copyright notice.

File: cscs-checks/tools/profiling_and_debugging/README.md (hunk 1 above)
Comment (jgphpc) [reply cscs-checks/tools/profiling_and_debugging/README.md]: I'll move as many as possible to the new repo

**Comment by vkarak** [comment]
@jgphpc We cannot merge this until you fix the conflicts. They are due to the fact that we have changed the files in the master (added a copyright notice) and you are now removing them.

**Comment by jgphpc** [comment]
@vkarak tests 🆗 on [daint](https://jenkins.cscs.ch/blue/organizations/jenkins/reframe-daint-hpctools/detail/reframe-daint-hpctools/12/pipeline/)  (and dom)

**Pull request merged**